src/hardware/openbench-logic-sniffer/api.c

```diff
@@ -47,6 +48,12 @@ static const int32_t trigger_matches[] = {
 	SR_TRIGGER_ONE,
 };
 
+static const char* external_clock_edges[] = {
+	"r", // positive edge
+	"f"  // negative edge
+};
+
+
```

Comment:
Superfluous empty line. Other than that, the patch looks good and I can't say much more. I don't have and never used an external clock. But it is surely worth to have.

Reply:
Thank you! Will remove the empty line.